src/App.js

```diff
@@ -1,15 +1,86 @@
 import React from 'react';
+import TodoList from './components/TodoComponents/TodoList';
+import TodoForm from './components/TodoComponents/TodoForm';
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor(props) {
+    super(props)
+      this.state = { list: [
+        {item: 'walk dog',
+         id: 1,
+         complete: false},
+        {item: 'dishes',
+         id: 2,
+         complete: false},
+        {item: 'laundry',
+         id: 3,
+         complete: false}
+      ],
+      newItem: '',
+      newThing: 1, 
+
+      }
+  }
+
+  crossOut = (index) => {
+    this.setState({list: this.state.list.map((item, idx) => {
+      if(index === idx) {
+        return {
+          ...item,
+          complete: item.complete === false ? true : false, 
```

Comment:
Another possibility here would be to use a toggler like `complete: !item.complete`